draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1124,28 +1132,30 @@ Output: (x, y), a point on E.
 
 Constants:
 1.  c1 = -B / A
-2.  c2 = Z
-3.  c3 = sqrt(-Z^3)
+3.  c2 = -1 / Z
```

Comment:
nit: what happened to constant #2?

Reply:
It looks like in the other impls, named constants are just used directly. In other words, rather than using `c2 = Z`, other impls just use `Z` directly. I did the same here, which killed the old `c2`, and just forgot to renumber.

Do we want to revisit this style decision throughout?

Reply:
Ah, sorry, I meant that the list of constants is missing a second item! It has #1 and #3.

Reply:
Got it. Thanks, fixed!